components/Nav/nav.js

```diff
@@ -48,6 +47,8 @@ export default function NavBar() {
                         <Link
                             href={item.href}
                             key={item.name}
+                            target={item.target}
```

Comment:
Why are these being added?
These are the wrong props since this is not traditional HTML - it is a NextJS component
https://nextjs.org/learn/basics/navigate-between-pages/link-component

Reply:
This was branched off from the other PR for fundraising page, so I didn't add those in. I'll checkout that link though.